src/server/game/Entities/Creature/CreatureData.h

```diff
@@ -447,7 +447,7 @@ struct TC_GAME_API CreatureBaseStats
 
     uint32 GenerateHealth(CreatureTemplate const* info) const
     {
-        return uint32(ceil(BaseHealth[info->HealthScalingExpansion] * info->ModHealth * info->ModHealthExtra));
+        return uint32(std::lroundf(BaseHealth[info->HealthScalingExpansion] * info->ModHealth * info->ModHealthExtra));
```

Comment:
ceil was used here, because some npcs have have very low ModHealth and to prevent generated health is 0

Reply:
Then only 0 should be changed to 1. But rounding is always the case: https://docs.google.com/spreadsheets/d/12Tevx7OKlQqEOLe8dXJ8iBo47a_4YMx-W1A4P-5KEFI/edit?usp=sharing

Reply:
either way this is unrelated to powers - shouldnt be here